application/src/main/java/org/opentripplanner/routing/algorithm/mapping/RaptorPathToItineraryMapper.java

```diff
@@ -82,7 +83,8 @@ public RaptorPathToItineraryMapper(
     TransitService transitService,
     RaptorTransitData raptorTransitData,
     ZonedDateTime transitSearchTimeZero,
-    RouteRequest request
+    RouteRequest request,
+    OsmStreetDecoratorService osmStreetDecoratorService
```

Comment:
When adding something to a list - look at the list an try to see if there is a convention and follow it. The services are listed first here, and more volatile objects at the end. Place your `osmStreetDecoratorService` after the `raptorTransitData`.